biztalk/core/how-to-modify-group-settings.md

```diff
@@ -38,10 +38,19 @@ Using the Settings Dashboard, you can modify the configuration information used
    | **Message box performance counter sampling interval** |                                                                       Set the interval at which performance counters are refreshed.<br /><br /> The interval trades off load on database versus up-to-dateness of counters. The higher value means less frequently updated data, and thus lesser load on the database.                                                                        | 1 – Maximum value of type Integer |       -       |               Largest value on any machine in the BizTalk group if present. If not, default.                |
    |            **Enable group level tracking**            | Select this option to turn the group level tracking for BizTalk Server on.<br /><br /> Turning off global tracking disables the tracking interceptors for the entire [!INCLUDE[btsBizTalkServerNoVersion](../includes/btsbiztalkservernoversion-md.md)] group. This means, BizTalk Server will not track events in its tracking tables. **Note:**  This setting does not impact BAM Tracking. |              On, Off              |      On       |                                                      -                                                      |
-
+   |            **Allow import of tracking settings**      | New starting with BizTalk Server 2026.<br /><br /> Select this option to turn the group level tracking for BizTalk Server on.<br /><br /> Turning off global tracking disables the tracking interceptors for the entire [!INCLUDE[btsBizTalkServerNoVersion](../includes/btsbiztalkservernoversion-md.md)] group. This means, BizTalk Server will not track events in its tracking tables. **Note:**  This setting does not impact BAM Tracking. |              On, Off              |      On       |
+   |            **Audit management Operations**            | New starting with BizTalk Server 2020.<br /><br />Select this option to enable audit of management operations.<br /><br /> |              On, Off              |      Off       |
+   |            **Maximum number of audit entries**        | New starting with BizTalk Server 2020.<br /><br />Set the maximum number of audit logs to be retained in  audit store. BizTalk will automatically delete oldest records when number of audit entries exceeds this number. | 1 – Maximum value of type Integer |       10000       |              -               |
+   |             **Analytics**                             |
+   |            **Enable group-level analytics**           | New starting with BizTalk Server 2016 feature pack.<br /><br />Select this option to leverage the power of Azure (Application Insights and Azure Event Hubs) to monitor your BizTalk applications.  |              On, Off              |      Off       |
```

Comment:
I think you can add a link to the analytics page for all 3 analytics related settings (Enable, Target type, Connection parameters). 
Suggested text:

See [Send BizTalk Server tracking data to Azure](../core/send-tracking-data-to-azure-application-insights-using-biztalk-server.md) for more details.